web/themes/custom/novel/templates/views/node--event-list-teaser.html.twig

```diff
@@ -0,0 +1,31 @@
+<a href="{{ content.event_link }}" class="event-list-item arrow__hover--right-small">
+  <div class="event-list-item__image-container">
+    {% if content.field_event_image['#items'] is not empty %}
+      {% set image_uri = content.field_event_image['#items'].entity.field_media_image.entity.fileuri %}
+      {% set image_url = file_url(image_uri) %}
+        <img src="{{ image_url }}" alt="{{ content.label }}" class="event-list-item__image" />
+    {% endif %}
+  </div>
+  <div class="event-list-item__content">
+    <div aria-pressed="false" class="tag tag--fill cursor-pointer tag--small event-list-item__tag">foredrag</div>
```

Comment:
Consider removing this element entirely.

We have not implemented the correct value to display here so we might as well leave it out if we do not have something to display. The primary reason for leaving it in would be if keeping it out meant that the layout would be messed up.

Reply:
I see your point. 

Since the tag was included in the design system, i feel like it makes sense to also have it here, if not for anything else then just for the visual demonstration.  Retaining it in the template makes it easier to see the layout as a whole instead of having missing parts in my opinion 🥳